CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 ## Alloy Release Notes
 
+### Release 1.8.0
```

Comment:
We haven't been using `CHANGELOG.md` anymore between 1.7.7 en 1.7.35 and @jawa9000 is in the process of moving them to the [docs](http://docs.appcelerator.com/platform/latest/#!/guide/Alloy_Release_Notes).

It's fine, will delete it later.